presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -88,6 +91,11 @@ Property Name                                                 Description
 
 ``cassandra.split-size``                                      Number of keys per split when querying Cassandra.
 
+``cassandra.splits-per-node``                                 Number of splits per node. By default, the values from the
+                                                              ``system.size_estimates`` table are used. Only override when
+                                                              connecting to Cassandra versions < 2.1.5, which lack
```

Comment:
- typo: `lacks`
- can you add this explanation to cmt msg too? Otherwise it's unclear why support for protocol v2 relates to max splits per node.